cis/validation.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+
+from jsonschema import validate as jsonschema_validate
+
+from cis.encryption import decrypt
+
+
+CIS_SCHEMA_JSON = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'schema.json')
+
+
+with open(CIS_SCHEMA_JSON, 'r') as schema_data:
+    CIS_SCHEMA = json.load(schema_data)
+
+
+def validate(**payload):
+    """
+    Validates the payload passed to CIS.
+
+    :payload: Encrypted payload based on the output of `cis.encryption.encrypt` method
+    """
+
+    try:
+        # Decrypt payload coming from CIS using KMS key
+        # This ensures that publisher is trusted by CIS
+        decrypted_payload = decrypt(**payload)
+    except:
```

Comment:
Not really, I already though about it. The reasons are:

* The exceptions are like 20 (considering that its both AWS KMS API exceptions and AES cryptography exceptions)
* In all the cases we need to return ``False``.

What's missing but its outside of the scope is:

* Handle logs to cloudwatch or another log provider
* Pass it back to the stream (although this needs discussion and its completely outside the current OKR goal)